### PR TITLE
[11.0][MIG] purchase_minimum_amount

### PR DESCRIPTION
**Pull request by rgarnau**
no major changes.

This depends on:
- [X] purchase_order_block #480

### CODE REVIEW

**Review by lasley — APPROVED** [review]

**Comment by lasley** [comment]
👍  Code, no test

**Review by etobella — APPROVED** [review]
LGTM :+1:

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+purchase_order_block https://github.com/Eficent/purchase-workflow 11.0-mig-purchase_order_block-refactor
```
Comment (lasley) [inline oca_dependencies.txt]: What PR does this depend on?

Comment (rgarnau) [reply oca_dependencies.txt]: @lasley - This one: https://github.com/OCA/purchase-workflow/pull/480

Comment (lasley) [reply oca_dependencies.txt]: Thanks @rgarnau. In the future, please note this in your PR description in order to highlight the dependencies. I went ahead and did it here.

Comment (rgarnau) [reply oca_dependencies.txt]: That's true! Forgot! Thanks @lasley!

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional and technical review

**Pull request merged**